wiki/Tournaments/CWC/1/en.md

```diff
@@ -157,91 +157,322 @@ This competition has come to an end and resulted in the following podium:
+   - **A team should have at least three players registered in order to be eligible to participate.**
+4. Once a team roster has been accepted, it remains locked for the rest of the tournament. No further changes may be made to it except on extraordinary circumstances under the Tournament Management's approval.
+
+### Match instructions
+
+1. A member of the Tournament Management will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'Score'. Room name must be "CWC: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. As the Catch the Beat World Cup features no formal referee, **both team captains are expected to referee the match by themselves from inside the multiplayer lobby**. If there are any suspected irregularities during the run of the match, the concerned team captain may file a report in the discussion thread which will be investigated by the Tournament Management in due time.
+3. Map banning **does not apply** in the entirety of the tournament.
+4. Each team must have three players for each map. They can be exchanged freely after a map is concluded.
+5. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
+6. Each captain must use the `!roll` command once in `#multiplayer`.
+   - The captain with the higher roll decides which team **picks** first.
+7. The results of each match will be posted on the discussion thread after the match has been concluded by either of the team captains.
+
+### Stage instructions
+
+1. In the Group Stage, the teams will be divided into four groups of five teams each.
+2. All the teams from each group will face each other over the course of five weeks.
+   - Due to the odd number of teams in each group, all teams **will only be playing on four matchweeks** instead of on all five by default. Team captains are expected to be aware of the schedules announced by the Tournament Management.
+3. Depending on the outcome of a Group Stage match, each team will be alloted Group Stage points to their name as follows:
+   - Winning a match gives the team +3 Group Stage points.
+   - Losing a match gives the team +1 Group Stage point.
+   - Forfeiting a match, either by disqualification or by a Lost by Default, penalizes the team by -3 Group Stage points.
+4. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most Group Stage points accumulated.
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+     - Have higher `∑{(total score difference) / (maximum score)}`.
+     - Winner of the rematch.
```

Comment:
these should probably be numbered. not sure if this will make remark angry tho
```suggestion
   1. Most Group Stage points accumulated.
   2. Most matches won.
   3. Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
   4. Most beatmaps won.
   5. **Winner of the match played previously between the tied teams.**
   6. In the event of a triple tie, the following priority will take place:
     1. Have higher `∑{(total score difference) / (maximum score)}`.
     2. Winner of the rematch.
```

Reply:
i'm quite sure this will make the remark angry, yeah :(

either way all other tournament articles that feature Group Stage (esp. world cups) are writing this in list format iirc so yeah i'm keeping this as well

![image](https://user-images.githubusercontent.com/36564236/106345945-150b6300-62e6-11eb-8ce2-8b2a77e4043c.png)